src/vendors/VendorsScreen.tsx

```diff
@@ -1,27 +1,39 @@
 import { map } from 'lodash'
-import React, { useState } from 'react'
-import { StyleSheet } from 'react-native'
+import React, { useEffect, useRef, useState } from 'react'
+import { RefreshControl, RefreshControlProps, StyleSheet } from 'react-native'
 import { FlatList } from 'react-native-gesture-handler'
 import Animated from 'react-native-reanimated'
 import { SafeAreaView } from 'react-native-safe-area-context'
+import { useDispatch } from 'react-redux'
 import { navigate } from 'src/navigator/NavigationService'
 import { Screens } from 'src/navigator/Screens'
+import useSelector from 'src/redux/useSelector'
+import colors from 'src/styles/colors'
+import { fetchVendors } from 'src/vendors/actions'
 import { Vendor } from 'src/vendors/types'
 import VendorDetailBottomSheet from 'src/vendors/VendorDetailBottomSheet'
 import VendorListItem from 'src/vendors/VendorListItem'
-import { CuracaoVendors } from 'src/vendors/vendors'
 
 const AnimatedFlatList = Animated.createAnimatedComponent(FlatList)
 
 export default function VendorsScreen() {
-  const sections = map(CuracaoVendors, (vendor: Vendor) => {
+  const dispatch = useDispatch()
+  const vendors = useSelector((state) => state.vendors.vendors)
```

Comment:
Create a selector in the `vendors/actions.ts` for commonly used selectors, e.g.
`const vendorsSelector = (state: RootState) => state.vendors.allVendors`